docs/proposals/algorithms/lifelong-learning/Unknown Task Recognition Algorithm Reproduction based on Lifelong Learning of Sedna.md

```diff
@@ -0,0 +1,139 @@
+# Unknown Task Recognition Algorithm Reproduction based on Lifelong Learning of Sedna
+Traditional machine learning performs test-set inference by training known samples to which its knowledge is limited. The model which is trained by traditional machine learning cannot be effectively recognized the unknown samples from new classes and they will be processed as known samples. Therefore, the recognition and processing of unknown samples or unknown tasks will become the main research direction of artificial intelligence in the future. This project aims to reproduce the CVPR2021 paper "Learning placeholders for open-set recognition" in the Semantic Segmentation dataset. This paper proposes placeholders that imitate the emergence of new classes, thus helping to transform closed-set training into open-set training.
+
+
+
+## Goals
+1. The reproduction should be completed based on the Cityscapes Semantic Segmentation dataset.
+2. The reproduced code is successfully merged into the lifelong learning module of Sedna.
```

Comment:
Well, not Sedna here and should be Ianvs. Besides, the algorithm description is also needed, e.g., [Incremental learning: BasicIL-FPN](https://ianvs.readthedocs.io/en/latest/proposals/algorithms/incremental-learning/basicIL-fpn.html).

Reply:
Please follow up this comment.

Reply:
Received and completed.

Reply:
ok